projects/maiar/index.js

```diff
@@ -1,32 +1,35 @@
 const { request, gql } = require("graphql-request");
 const { toUSDTBalances } = require('../helper/balances')
-const { sumTokensExport } = require('../helper/sumTokens')
 
 const LiquidityQuery = gql`
 {
-    factory {
-      pairCount
-      totalValueLockedUSD
-    }
+    totalValueLockedUSD
+  }
+`
+
+const StakeQuery = gql`
+{
+    totalValueStakedUSD
```

Comment:
hi @dragos-rebegea how is this computed? according to coingecko, marketcap of MEX is 81M, but this is twice that